# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/generic/HyperTextAccessible-inl.h
# Commit: 5e92b9c7d479
# Full Hash: 5e92b9c7d4794af1d7ab0fb5c278c2e681a18fc6
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1811092
# File Overlap Count: 1
# Description:
#   Bug 1811092 - P1: Add new async add/set text selection message. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D167201
# ==============================================================================

diff -r 7cebaa85f4ff -r 5e92b9c7d479 accessible/generic/HyperTextAccessible-inl.h
--- a/accessible/generic/HyperTextAccessible-inl.h	Fri Jan 27 20:03:06 2023 +0000
+++ b/accessible/generic/HyperTextAccessible-inl.h	Fri Jan 27 20:15:11 2023 +0000
@@ -25,13 +25,6 @@
   SelectionMgr()->UpdateCaretOffset(this, aOffset);
 }
 
-inline bool HyperTextAccessible::AddToSelection(int32_t aStartOffset,
-                                                int32_t aEndOffset) {
-  dom::Selection* domSel = DOMSelection();
-  return domSel &&
-         SetSelectionBoundsAt(domSel->RangeCount(), aStartOffset, aEndOffset);
-}
-
 inline void HyperTextAccessible::ReplaceText(const nsAString& aText) {
   if (aText.Length() == 0) {
     DeleteText(0, CharacterCount());